symbology/unique-value-renderer/src/main/java/com/esri/samples/unique_value_renderer/UniqueValueRendererSample.java

```diff
@@ -62,8 +63,12 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create ArcGISMap with topographic basemap
-      final ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+      // create ArcGISMap with topographic basemap style
```

Comment:
```suggestion
      // create map with the topographic basemap style
```

Reply:
Made a few changes to this one on the `Viewpoint` side of things too